user_stories.md

```diff
@@ -0,0 +1,20 @@
+Marketing Research Team:
```

Comment:
Try to write these in the style of the other user stories we've seen: "As a X, I want Y in order to Z."